pyvista/utilities/_getattr.py

```diff
@@ -0,0 +1,57 @@
+import importlib
+import inspect
+import pathlib
+import warnings
+
+# Have to set __path__ to this file's directory so that
+# `import pyvista.utilities` imports this directory
+__path__ = [str(pathlib.Path(__file__).parent)]
+
+# Places to look for the utility
+_MODULES = [
+    'pyvista.core.utilities',
+    'pyvista.core.errors',
+    'pyvista.core',
+    'pyvista.plotting.utilities',
+    'pyvista.plotting.errors',
+    'pyvista.plotting.texture',
```

Comment:
I set the order intentionally because everything that is available under `pyvista.core.utilities` and `pyvista.core.errors` is also available under `pyvista.core` (due to `*` imports and wanting to keep the `pyvista.*` namespace the same as previous releases). Having this specific order will provide better deprecaton notice and tell users to import from `pyvista.core.utilities` specifically as opposed to just `pyvista.core` for everything (same goes for `pyvista.plotting`)

In the way I currently have it configured, `pyvista.core` is a fallback if something moved that isn't in `pyvista.core.utilities` or `pyvista.core.errors` which there are several things.

Reply:
Ah, good point, I didn't realise that the order mattered here. Thanks.